indexer/feature.hpp

```diff
@@ -116,6 +116,11 @@ class FeatureType
   }
 
   size_t GetPointsCount() const;
+  size_t GetTrianglesCount(int scale)
```

Comment:
its not count of triangles, but a count of all points making triangles (hence x3), right?

Reply:
Yes, will rename.